### DISCUSSION TITLE
Removing Systems Screen

### DISCUSSION THREAD

**Discussion by alash3al** [orchidsoftware/platform#1608] [Q&A] [answered]
I just noticed that the systems screen has been removed from the master branch, Is that true? or there will be some re-arrangement because systems screen contains users/roles screens which give orchid more power over competitors.

**Comment by tabuna** [comment, marked as answer]
Hi, user and role management won't be removed for sure. At the moment, we have only removed the menu because many users did not understand that there is another menu hidden from the eyes, and also, not all projects need so many menus (they simply have nothing to fill them with)
